#request
string vision_label
---
#response
float32 angle
